controllers/etcd_controller_test.go

```diff
@@ -600,6 +604,63 @@ var _ = Describe("Druid", func() {
 		Entry("if the store is Swift, the statefulset should reflect the spec changes", "foo56", getEtcdWithSwift, validateStoreOpenstack),
 		Entry("if the store is OSS, the statefulset should reflect the spec changes", "foo57", getEtcdWithOSS, validateStoreAlicloud),
 	)
+
+	DescribeTable("when etcd resource is created with compactionScheule field",
```

Comment:
Thanks for adding the test case @aaronfern! Can you please add two more cases?
1. If compaction schedule is not set then the cronjob should not be created.
2. If compaction schedule is not set and a cronjob already exists then the cronjob should be deleted.

Reply:
Added these two test cases